internal/pkg/deploy/cloudformation/cloudformation.go

```diff
@@ -237,31 +304,21 @@ func (cf CloudFormation) createChangeSet(in *cloudformation.CreateChangeSetInput
 	}, nil
 }
 
-// stackExists returns true if the underlying error is a stack already exists error.
-func stackExists(err error) bool {
-	currentErr := err
-	for {
-		if currentErr == nil {
-			break
-		}
-		if aerr, ok := currentErr.(awserr.Error); ok {
-			switch aerr.Code() {
-			case "ValidationError":
-				// A ValidationError occurs if we tried to create the stack with a change set.
-				if strings.Contains(aerr.Message(), "already exists") {
-					return true
-				}
-			case cloudformation.ErrCodeAlreadyExistsException:
```

Comment:
is this not a thing?

Reply:
Na - we're doing the check preemptively, rather than parsing the response of the change set creation error.